lib/dockerregistry/inventory.go

```diff
@@ -1033,6 +1022,22 @@ func (sc *SyncContext) ReadRegistries(
 	sc.ExecRequests(populateRequests, processRequest)
 }
 
+func SplitByKnownRegistries(
+	r RegistryName,
+	rcs []RegistryContext) (RegistryName, ImageName, error) {
+
+	for _, rc := range rcs {
+		//		if inameIdx := strings.LastIndex(string(r), string(rc.Name)); inameIdx != -1 {
```

Comment:
Probably should remove this dead code (not necessarily in this PR)

Reply:
ACK